grayskull/pypi/pypi.py

```diff
@@ -787,15 +809,23 @@ def _extract_requirements(self, metadata: dict) -> dict:
         :return: all requirement section
         """
         name = metadata["name"]
-        requires_dist = PyPi._format_dependencies(metadata.get("requires_dist"), name)
-        setup_requires = (
-            metadata.get("setup_requires") if metadata.get("setup_requires") else []
-        )
-        host_req = PyPi._format_dependencies(setup_requires, name)
+        if metadata.get("requires_dist"):
+            requires_dist = PyPi._format_dependencies(metadata.get("requires_dist"), name)
+        else:
+            requires_dist = metadata.get("requires_dist")
+
+        if metadata.get("setup_requires"):
+            setup_requires = (
+                metadata.get("setup_requires") if metadata.get("setup_requires") else []
```

Comment:
This can be expressed more succintly like this:

```suggestion
                metadata.get("setup_requires", [])
```

`{}.get()` accepts a second parameter, which will be the value returned if the requested key was not found.